server/repository/src/db_diesel/changelog/changelog.rs

```diff
@@ -77,6 +78,8 @@ pub enum ChangelogTableName {
     AssetCategory,
     AssetType,
     AssetCatalogueItem,
+    #[default]
+    SyncFileReferenceRow,
```

Comment:
I'm unsure why the naming differs from the others.. Does the `Row` mean anything? 
Is this just placeholder code? can't see the corresponding usage in the translations file

Reply:
It's just a mistake, thanks for noticing!